src/NUnitEngine/nunit.engine.tests/Internal/DirectoryFinderTests.cs

```diff
@@ -455,9 +455,13 @@ public void GetDirectories_PatternIsNull()
         [Test]
         public void GetDirectories_PatternIsEmpty()
         {
-            var finder = new DirectoryFinder(Substitute.For<IFileSystem>());
+            var directory = Substitute.For<IDirectory>();
+            var sut = new DirectoryFinder(Substitute.For<IFileSystem>());
+
+            var directories = sut.GetDirectories(directory, string.Empty);
 
-            Assert.That(() => finder.GetDirectories(Substitute.For<IDirectory>(), string.Empty), Throws.ArgumentException.With.Message.Contains(" pattern "));
+            Assert.That(directories, Has.Count.EqualTo(1));
+            Assert.That(directories.First(), Is.EqualTo(directory));
```

Comment:
Spotted this one above - out of interest, why the change in behaviour here? Not saying I disagree with it - I just don't understand why? 😄

Reply:
An empty pattern is used when a directory is specified in the addins-file. I think to express "do not scan for subdirectories",  an empty pattern is a better way than using `null`.

The using code in `ExtensionService` is:
```c#
if (path.EndsWith("/"))
{
  return new Backports.Tuple<IDirectory, string>(_fileSystem.GetDirectory(path), string.Empty);
}
```